k8s/upgrades/jiva-0.6.0-0.8.1/README.md

```diff
@@ -0,0 +1,151 @@
+# UPGRADE FROM OPENEBS 0.6.0 TO 0.8.1
+
+## Overview
+
+This document describes the steps for upgrading OpenEBS from 0.6.0 to 0.8.1 
+
+The upgrade of OpenEBS is a two step process: 
+- *Step 1* - Upgrade the OpenEBS Operator 
+- *Step 2* - Upgrade the OpenEBS Volumes from previous versions 0.6.0 
+
+### Terminology
+- *OpenEBS Operator : Refers to maya-apiserver & openebs-provisioner along w/ respective services, service a/c, roles, rolebindings*
+- *OpenEBS Volume: The Jiva controller(aka target) & replica pods*
+
+## Prerequisites
+
+*All steps described in this document need to be performed on the Kubernetes master or from a machine that has access to Kubernetes master*
+
+### Download the upgrade scripts
+
+You can either `git clone` or download the upgrade scripts.
+
+```
+mkdir upgrade-openebs
+cd upgrade-openebs
+git clone https://github.com/openebs/openebs.git
+cd openebs/k8s/upgrade/jiva-0.6.0-0.8.1/
+```
+
+### Breaking Changes in 0.8.1
```

Comment:
leave it as it is - as you copied from original upgrade script

Reply:
done